lib/mayaUsd/render/vp2RenderDelegate/mesh.cpp

```diff
@@ -1933,9 +1937,12 @@ void HdVP2Mesh::_UpdateDrawItem(
             if ((colorInterp == HdInterpolationConstant || colorInterp == HdInterpolationInstance)
                 && (alphaInterp == HdInterpolationConstant
                     || alphaInterp == HdInterpolationInstance)) {
-                const GfVec3f& clr3f = MayaUsd::utils::ConvertLinearToMaya(colorArray[0]);
+                const GfVec3f& clr3f = colorArray[0];
                 const MColor   color(clr3f[0], clr3f[1], clr3f[2], alphaArray[0]);
                 shader = _delegate->GetFallbackShader(color);
+                if (shader) {
+                    shader->setParameter("diffuse", 1.0f);
```

Comment:
We already multiply the exported display color by the `diffuse` coefficient of 0.8. We need not apply it a second time.